november-2017/php-version.md

```diff
@@ -0,0 +1,6 @@
+### Upcoming MyBB Releases And Their Minimum Supported PHP Versions
+Frequent visitors to the MyBB Community Forums may remember [a discussion from July 2016](https://community.mybb.com/thread-195675.html) regarding the minimum PHP version that MyBB 2.0 would require. The intentions were that MyBB 2.0 would require PHP 7.0 and other environment requirements could be changed with each minor release (for example from 2.3 to 2.4).
+
+With the recent discussion of the 1.10 series and the uncertainty surrounding *exactly* what is happening to MyBB 2.0, we believe that changes could and *should* happen as soon as 1.10. At the time of writing this blog post, the current stable release of MyBB is 1.8.13 and the minimum supported PHP version is 5.2. To put that into some perspective: PHP 5.2 was released on 2nd November, 2006 and was then declared end of life (EOL: meaning it is unmaintained and will have no more security fixes) on 6th January, 2011. PHP 5.2 has served us well but has not received any performance or security updates in over 6 years, it actively hinders the development and modernisation of MyBB as a whole. Since then there have been several other PHP releases both made generally available and since reaching end of life status. The PHP team maintain [a list of versions that have reached end of life](https://secure.php.net/eol.php) status that is kept up to date and provides exact dates for version lifetimes.
```

Comment:
1.8.14

Reply:
Suggested:
```diff
- end of life (EOL: meaning it is unmaintained and will have no more security fixes)
+ End Of Life (meaning it will not receive any updates nor security fixes)
```